packages/@aws-cdk/aws-eks/lib/private/service-available.ts

```diff
@@ -0,0 +1,10 @@
+import { Token } from '@aws-cdk/core';
+import { RegionInfo } from '@aws-cdk/region-info';
+
+export function EcrPublicAvailable(region: any): boolean {
+  if (region && !Token.isUnresolved(region)) {
```

Comment:
Is there any handling we can do for when a token is being used? It seems like it's a fairly common use case and this fix would still result in an error for those users.

Reply:
Yes thank you for the heads up. I will fix this soon.